ncids-dart-sass/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "dart-sass",
+	"version": "1.0.0",
+	"description": "NCIDS example repository using dart sass",
+	"main": "index.js",
+	"keywords": [
+		"National Cancer Institute Design System",
+		"ncids",
+		"sass",
+		"css"
+	],
+	"author": "National Cancer Institute",
+	"license": "ISC",
+	"scripts": {
+		"build:css": "sass -I ./node_modules/@nciocpl/ncids-css/uswds-packages -I ./node_modules/@nciocpl/ncids-css/packages ./src/index.scss ./dist/ncids-compiled.css && postcss ./dist/ncids-compiled.css --output ./dist/styles.css",
```

Comment:
This can be:
`sass -I ./node_modules/@nciocpl/ncids-css/uswds-packages -I ./node_modules/@nciocpl/ncids-css/packages ./src/index.scss | postcss  --output ./dist/styles.css`

Sass will output to standard out when not given an output file name. Postcss will read from standard in when you have not specified a filename.

The same can apply below. This would also need some info on the readme.